content/en/monitors/service_level_objectives/guide/slo_graph_query.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Scope metric-based SLO queries
+kind: guide
+disable_toc: false
+further_reading:
+- link: "/dashboards/widgets/slo/"
+  tag: "Documentation"
+  text: "SLO Summary Widget"
+---
+
+<div class="alert alert-info">This feature is only available for <strong>metric-based</strong> SLO queries.</div>
+
+## Overview
+
+The [SLO Summary widget][1] supports advanced metric query filtering, including the use of template variables to dynamically scope results displayed. 
+
+## Walk through of an SLO query
+
+### Metric-based SLO query
+Create a [metric-based SLO][2]. This example uses APM trace metrics to measure the availability of an example service called `web-store`.
+
+##### Good events (numerator)
```

Comment:
```suggestion
#### Good events (numerator)
```

Reply:
This is a formatting thing, I just liked the purple headers over the black